src/index.js

```diff
@@ -13,6 +13,12 @@ const minStorybookSemver = '>=3'
 const v3StorybookSemver = '^3'
 const v4StorybookSemver = '^4'
 
+const ensureArray = (value) => {
+  if (value instanceof Array) return value
```

Comment:
this should be `Array.isArray`. don't ask me why it's better as i have no idea haha 😄 i just know it's preferred in the js world from what i've seen